deltachat-ios/AppDelegate.swift

```diff
@@ -49,7 +49,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         // setupCrashReporting() may create an additional handler, but we do not want to rely on that
         signal(SIGPIPE, SIG_IGN)
 
-        bgIoTimestamp = Double(Date().timeIntervalSince1970)
+        self.pushToDebugArray("🏁")
```

Comment:
setting `bgIoTimestamp` here seems to be wrong in case the app was not freezed by actually killed (what happens more often on older phones than on newer)